File: src/com/l2kt/gameserver/network/clientpackets/RequestOustPartyMember.java
package com.l2kt.gameserver.network.clientpackets;

import com.l2kt.gameserver.model.actor.instance.Player;
import com.l2kt.gameserver.model.group.Party;

public final class RequestOustPartyMember extends L2GameClientPacket
{
	private String _name;
	
	@Override
	protected void readImpl()
	{
		_name = readS();
	}
	
	@Override
	protected void runImpl()
	{
		final Player player = getClient().getActiveChar();
		if (player == null)
			return;
		
		final Party party = player.getParty();
		if (party == null || !party.isLeader(player))
			return;
		
		party.removePartyMember(_name, Party.MessageType.EXPELLED);
	}
}

File: src/com/l2kt/gameserver/network/clientpackets/RequestOustPartyMember.kt
package com.l2kt.gameserver.network.clientpackets

import com.l2kt.gameserver.model.group.Party

class RequestOustPartyMember : L2GameClientPacket() {
    private var _name: String = ""

    override fun readImpl() {
        _name = readS()
    }

    override fun runImpl() {
        val player = client.activeChar ?: return

        val party = player.party
        if (party == null || !party.isLeader(player))
            return

        party.removePartyMember(_name, Party.MessageType.EXPELLED)
    }
}